test.spec.js

```diff
@@ -0,0 +1,33 @@
+
+const fs = require('fs');
+const expect = require('expect');
+const { readFile, getLinks, getUri } = require('./src/app.js');
+const { validateLinks } = require('./src/main.js');
+
+let content = fs.readFileSync('./files/links.md', 'utf8');
+let path = './files/links.md'
+let badPath = './app.js'
+
+describe('readFile', () => {
+	it('Should be a function', () => {
+		expect(typeof readFile).toBe('function');
+	});
+});
+
```

Comment:
puedes tener el siguiente test 

```suggestion
describe('readFile', () => {
	it('Get all text of the file', () => {
	       let functionReadFile=readFile("README.md")
		expect(typeof functionReadFile).toBe('string');
	});
});
```